exalearn-ui/src/app/components/data-table/data-table.component.html

```diff
@@ -46,7 +46,7 @@
 
 		<ng-container matColumnDef="assigned by">
 			<th mat-header-cell *matHeaderCellDef mat-sort-header>{{ 'TABLES.ASSIGNED_BY' | translate}}</th>
-			<td mat-cell *matCellDef="let element">{{element.assigner}}</td>
+			<td mat-cell *matCellDef="let element">{{element.assigner || element.assignedBy}}</td>
```

Comment:
Why two different keys here?
Is it expected?

Reply:
Added TODO